pkg/config/config.go

```diff
@@ -250,13 +250,13 @@ func (c *ScrapeConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 	// Validate the scrape and timeout internal configuration. When /debug/pprof/profile scraping
 	// is enabled we need to make sure there is enough time to complete the scrape.
```

Comment:
This comment would be wrong

Reply:
How so? Because it's not only applying this for `/profile`?

Reply:
Sorry, I miss read, bad case of Mondays :sweat_smile: